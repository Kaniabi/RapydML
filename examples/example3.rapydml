div(id="content"):
  "I haven't yet put anything on this page"
  p:
    "Compatible with all major browsers:"
    for $item in [[firefox, Firefox], [chrome, Chrome], [ie, Internet Explorer], [opera, Opera], [safari, Safari]]:
      img(src="$item[0].jpg", alt="$item[1]")
