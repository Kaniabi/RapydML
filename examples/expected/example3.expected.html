<div id="content">
  I haven't yet put anything on this page
  <p>
    Compatible with all major browsers:
    <img src="firefox.jpg" alt="Firefox" />
    <img src="chrome.jpg" alt="Chrome" />
    <img src="ie.jpg" alt="Internet Explorer" />
    <img src="opera.jpg" alt="Opera" />
    <img src="safari.jpg" alt="Safari" />
  </p>
</div>
